src/FSharp.Data.JsonValidation/Library.fs

```diff
@@ -58,6 +66,16 @@ module public JsonValidation =
         then stringMeetsProperties str rest
         else Invalid <| sprintf "Expected string %A to %s" str description
 
+  let rec private numberMeetsProperties num props =
+    match num, props with
+      | _, [] -> Valid
+      | num, IsPositive::rest when num > (decimal 0) -> numberMeetsProperties num rest
```

Comment:
I suspect `0M` will be slightly more performant than `decimal 0` because `decimal 0` has to convert an int to the decimal which is always the same value.